tools/load-examples.groovy

```diff
@@ -1,341 +0,0 @@
-import org.ow2.proactive.authentication.ConnectionInfo
```

Comment:
ideally, we should see in the pull-request the renaming from load-examples.groovy, to LoadPackage.groovy, the PR does not show this as a renaming but as a delete file and new file, making it difficult to track the changes